act/visit_details_map.sql

```diff
@@ -8,3 +8,144 @@ drop table "&&metadata_schema".NCATS_VISIT_DETAILS purge;
 whenever sqlerror exit sql.sqlcode;
 
 create table "&&metadata_schema".NCATS_VISIT_DETAILS  nologging as select * from "&&shrine_ont_schema".NCATS_VISIT_DETAILS ;
+
+
+-- remap visit types to use whatever is in heron_terms
+
+-- ambulatory visit
```

Comment:
This will work, but at  some point we need to have design discussion about why we don't populate `visit_dimension.inout_cd` for HERON. Maybe Dan would know. Why? @mwennberg :Do you know why?
 If we do that for HERON, we will not need to make changes to visit details for ACT. 

I would prefer visit related information comes from visit dimension.
``` 
select count(*) from BHDATAA1.visit_dimension
where inout_cd is not null;
--0
```

Reply:
@lav-patel I am unaware of the reason behind not populating inout_cd. If you plan to use the visit dimension - have you looked into it much to know how well/accurately other items are populated?

Reply:
Can you elaborate on other items ? 

Have you experienced issues with other aspects of visit dimension ?

I want to make sure that we follow the national standards in HERON, so we don't have to reimplement stuff for add-ons like (ACT)